transactions/signer_approve.cdc

```diff
@@ -19,9 +21,12 @@ transaction(treasuryAddr: Address, actionUUID: UInt64, message: String, keyIds:
       j = j + 1
     }
 
-    var isValid = action.verifySignature(acctAddress: signer.address, message: message, keyIds: _keyIds, signatures: signatures, signatureBlock: signatureBlock)
+    self.messageSignaturePayload = MyMultiSig.MessageSignaturePayload(
+      _signingAddr: signer.address, _message: message, _keyIds: _keyIds, _signatures: signatures, _signatureBlock: signatureBlock
+    )
+
   }
   execute {
-    
+    var isValid = self.action.signerApproveAction(_messageSignaturePayload: self.messageSignaturePayload)
```

Comment:
Is there a way to return the result of the transaction?
Or the better approach is to add a post block to check if the result is expected